src/common/translation/i18n/fi.json

```
@@ -480,17 +480,11 @@
       }
     },
     "preliminaryForm": {
-      "childBirthdate": {
+      "childBirthyear": {
         "input": {
-          "day": {
-            "placeholder": "pv"
-          },
-          "label": "Lapsen syntymäaika (pp.kk.vvvv)",
-          "month": {
-            "placeholder": "kk"
-          },
+          "label": "Lapsen syntymäaika (vvvv)",
           "year": {
-            "placeholder": "vuosi"
+            "placeholder": "Vuosi"
```

```suggestion
            "placeholder": "Lisää lapsen syntymävuosi"
```
i.e. "Enter child’s last name" in Finnish, to make the placeholder more similar to other placeholders in the UI